sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -201,6 +201,14 @@ object SQLConf {
     .stringConf
     .createOptional
 
+  val OPTIMIZER_PLAN_CHANGE_LOG_BATCHES = buildConf("spark.sql.optimizer.planChangeLog.batches")
```

Comment:
Is it ok to reuse `spark.sql.optimizer.planChangeLog.level` for this batch log?

Reply:
@maropu  yes I think its better to use the same log level,it will be better for analysis.

Reply:
@maropu  yes I think its better to use the same log level,it will be better for analysis.

Reply:
cc: @maryannxue

Reply:
@maropu  But I have once concern,what if someone uses custom Batch with any of the existing rule's name.

Reply:
As I understand this is not for the log level, rather a filter for specific the batches to log, i.e., a counterpart to "spark.sql.optimizer.planChangeLog.rules". Since the names of the batches are very different from those of the rules and feel a little "random", I don't think it's a good idea to reuse that conf. Moreover, it could be used as trick to turn off batch logging.